Src/CZICmd/DisplaySettingsHelper.h

```diff
@@ -90,6 +90,18 @@ class ChannelDisplaySettingsWrapper :public libCZI::IChannelDisplaySetting
 
         return false;
     }
+
+    bool TryGetChannelIdAndName(std::tuple<std::string, std::tuple<bool, std::string>>* channelIdAndName) const override
+    {
+        if (channelIdAndName != nullptr)
+        {
+            *channelIdAndName = this->chDisplSettings.channelIdAndName;
+
+            return true;
+        }
+
+        return false;
```

Comment:
I'd reckon this does not make much sense, I mean - the return value is always true (if `channelIdAndName` is non-null). So - if there is no way that this information is not valid, then I'd think there is no need for a "TryGet"-method.